gobblin-modules/google-ingestion/src/main/java/gobblin/ingestion/google/webmaster/GoogleWebMasterSource.java

```diff
@@ -47,7 +50,12 @@
    * Must Provide.
    * Provide the property site URL whose google search analytics data you want to download
    */
-  public static final String KEY_PROPERTY = SOURCE_GOOGLE_WEBMASTER_PREFIX + "property_url";
+  public static final String KEY_PROPERTY = SOURCE_GOOGLE_WEBMASTER_PREFIX + "property_urls";
+  /**
+   * Optional. Default to true.
+   * Determine whether to add source property as the last column to your configured schema
+   */
+  public static final String KEY_INCLUDE_SOURCE_PROPERTY = SOURCE_GOOGLE_WEBMASTER_PREFIX + "include_source_property";
```

Comment:
Do you also intend to include more columns to schema at runtime in future? If so, its better to move to a model where additional columns are added to schema via converter. If not, we can stick to what you have.